environment/aws/topology_setup/test_server_platforms/c_register.py

```diff
@@ -150,6 +297,60 @@ def platform(self) -> str:
         """
         return "c_android"
 
+    def cbl_filename(self, version: str) -> str:
+        return f"couchbase-lite-c-enterprise-{version}-android.zip"
+
+    def _download_cbl(self):
+        android_lib_dir = (
+            C_TEST_SERVER_DIR
+            / "platforms"
+            / "android"
+            / "app"
+            / "src"
+            / "main"
+            / "cpp"
+            / "lib"
+        )
+        header(f"Downloading CBL library {self.version}")
+        build = 0
+        version_parts = self.version.split("-")
+        if len(version_parts) > 1:
+            build = int(version_parts[1])
+
+        DOWNLOAD_DIR.mkdir(0o755, exist_ok=True)
+        download_file = DOWNLOAD_DIR / "framework.zip"
+        downloader = CBLLibraryDownloader(
+            "couchbase-lite-c",
+            self.cbl_filename(self.version),
+            version_parts[0],
+            build,
+        )
+        downloader.download(download_file)
+        shutil.rmtree(
+            IOS_FRAMEWORKS_DIR / "CouchbaseLite.xcframework", ignore_errors=True
```

Comment:
This is IOS framework. Probably from copy and paste.

Reply:
Good catch, fixed!